### DISCUSSION TITLE
ERROR: config_files_tmp is here...

### DISCUSSION THREAD

**Discussion by AllendePrieto** [alexisInSAR/EZ-InSAR#32] [Help]
Hi again,
after following the steps of the processing with ISCE step by step, when I request the interferogram stack processing, I have this error:
![image](https://user-images.githubusercontent.com/131383377/233638023-9bac0057-cfa7-4d49-a1b6-655ea9c2d559.png)
The  directories that I have are:
![image](https://user-images.githubusercontent.com/131383377/233638319-90026f63-0583-411b-8bab-d8dc1299a406.png)
And when I want to run StamPS processing:
![image](https://user-images.githubusercontent.com/131383377/233638755-82cc8aab-6c72-4e17-bfe8-7b978cf761b3.png)

**Comment by alexisInSAR** [comment]
Can you try to delete the configs_files_tmp directory? 

Alexis

**Comment by AllendePrieto** [comment]
Yes, and it returns me this error:

![image](https://user-images.githubusercontent.com/131383377/233645742-29b84d85-2e13-4b38-ba0d-0f380531020c.png)

**And when run again:**

![image](https://user-images.githubusercontent.com/131383377/233645840-825783c7-dcb2-4dd6-aede-326386b697d7.png)

**Comment by alexisInSAR** [comment]
There is an error with the directories. Please, close EZ-InSAr, delete: 
- config_files_tmp
- merged (if it is a symbolic link)
- merged_mintpy

And rerun EZ-InSAR.

**Reply by AllendePrieto** [reply]
Thanks!
The error disappears.  
I click in "Select the best reference date" in Interferogram stack option and my coarse networks of interferograms shows in the figure:
![image](https://user-images.githubusercontent.com/131383377/234832419-2d002f98-fabc-49b8-b0f5-64aaf2ac77ed.png)
and values resulting from the process:
![image](https://user-images.githubusercontent.com/131383377/234832731-caed4a1f-97f0-4e86-833a-544a9c8ef45f.png)
but when I click in "Continue to generate SI IW stack" with the similar parameters that manual shows:
![image](https://user-images.githubusercontent.com/131383377/234833579-6d4403d0-1d60-48ba-9241-b082157ce6ed.png)
I proved with different values but the results is the same.  Which is the "Index in position 2"? Is the Azimut_looks? because I proved with value 1 and the error is the same.

**Reply by AllendePrieto** [reply]
Hi again!
After deleting the directories "config_files_tmp" and "merged", (I don't have "merged_mintpy" directory), in the StaMPS Processing, when I click in "Crop the SLCs", shows:
![image](https://user-images.githubusercontent.com/131383377/234839334-c1a21355-7c92-47ba-85ea-ef6343a2fa57.png)

**Reply by alexisInSAR** [reply]
Hi, 

For the first issue, I think you want to compute all available pairs (max. 11 connections). Please use the parameter [all] for ncon.

For the second issue, the command: 
````
sudo chmod a+x /home/callende/tools insar/EZINSAR/EZINSAR BIN//Suppfunctions/run_SLCcropStack_mod.csh 
````
should be fine.

**Comment by AllendePrieto** [comment]
Hi Alex!,
I have tried, but I get the same error.
![image](https://user-images.githubusercontent.com/131383377/235084294-f500c2eb-ad74-4ed1-a9e0-a676293f844c.png)
For the second issue, I receive the following message:
![image](https://user-images.githubusercontent.com/131383377/235087518-66fc8240-9781-48b1-a767-a6236869ddcd.png)
The set of directories I have is the following:
![image](https://user-images.githubusercontent.com/131383377/235087921-565b7239-e919-4189-9d45-19e0a62f76bd.png)
The geom_reference directory is not contained in the merged directory, it is external to it.

**Comment by alexisInSAR** [comment]
Okay, the error is related to a previous error in the first processing (EZ-InSAR fails because of an inconsistency between directories and GUI inputs). Can you explain your processing: each processed steps?

**Reply by AllendePrieto** [reply]
Yes! I indicate the steps in the attached document.
[STEPS.pdf](https://github.com/alexisInSAR/EZ-InSAR/files/11353217/STEPS.pdf)

**Comment by alexisInSAR** [comment]
Please try this: 
1. Close EZ-InSAR
2. Delete the symbolic link _merged_
3. Delete the directory _merged_minpty_
4. Rename the directory _merged_stamps_ by _merged_
5. Open EZ-InSAR
6. Open your work directory
7. Click on SLC_stack, wait 2 seconds, click on Ifg stack, wait 2 seconds.
